app/helpers/turbo/frames_helper.rb

```diff
@@ -38,7 +38,12 @@ module Turbo::FramesHelper
   #
   #   <%= turbo_frame_tag(Article.find(1), Comment.new) %>
   #   # => <turbo-frame id="article_1_new_comment"></turbo-frame>
+  #
+  # Raises an +ArgumentError+ if called without a frame id.
+  #   <%= turbo_frame_tag(nil) %> # => ArgumentError: You must supply a frame id
   def turbo_frame_tag(*ids, src: nil, target: nil, **attributes, &block)
+    raise ArgumentError, "You must supply a frame id" if ids.all? { |id| id.blank? }
```

Comment:
Could this condition be abbreviated?

```suggestion
    raise ArgumentError, "You must supply a frame id" if ids.all?(&:blank?)
```